docs/metrics.md

```diff
@@ -137,12 +137,12 @@ All memory pool metrics have the following tags:
 
 #### Thread metrics
 
-| Metric name                  | Description |
-| ---------------------------- | ----------- |
-| `runtime.jvm.threads.daemon` | The current number of live daemon threads.
-| `runtime.jvm.threads.live`   | The current number of live threads including both daemon and non-daemon threads.
-| `runtime.jvm.threads.peak`   | The peak live thread count since the Java virtual machine started or peak was reset.
-| `runtime.jvm.threads.states` | The current number of threads per `state` (metric tag).
+| Metric name                  | Instrument | Description |
+| ---------------------------- | ---------- | ----------- |
+| `runtime.jvm.threads.daemon` | [Gauge][g] | The current number of live daemon threads.
+| `runtime.jvm.threads.live`   | [Gauge][g] | The current number of live threads including both daemon and non-daemon threads.
+| `runtime.jvm.threads.peak`   | [Gauge][g] | The peak live thread count since the Java virtual machine started or peak was reset.
+| `runtime.jvm.threads.states` | [Gauge][g] | The current number of threads per `state` (metric tag).
```

Comment:
```suggestion
| `runtime.jvm.threads.daemon` | [Gauge][g] | The current number of live daemon threads.
| `runtime.jvm.threads.live`   | [Gauge][g] | The current number of live threads, including both daemon and non-daemon threads.
| `runtime.jvm.threads.peak`   | [Gauge][g] | The peak live thread count since the Java virtual machine started or the peak was reset.
| `runtime.jvm.threads.states` | [Gauge][g] | The current number of threads per `state` (metric tag).
```